storage/replica_command.go

```diff
@@ -743,6 +743,19 @@ func (r *Replica) PushTxn(batch engine.Engine, ms *engine.MVCCStats, args proto.
 	if err != nil {
 		return reply, err
 	}
+	// There are three cases in which there is no transaction entry, in decreasing likelihood
+	// * a transaction is still active; it just hasn't committed/restarted/aborted/heartbeat yet
+	// * the transaction resolved its intents synchronously on successful commit; in this case,
+	// the transaction record is also removed. Note that in this case, the intent which prompted
+	// this PushTxn doesn't exist any more.
+	// * the transaction timed out/was aborted and the intent not cleaned up, but the transaction
+	// record recycled.
+	// We currently make no attempt at guessing which one it is, though we could (see #1939).
+	// Instead, a new entry is always written, and the Push assumes in all cases that the transaction is still running.
```

Comment:
s/and the Push assumes in all cases that the transaction is still running//
(if pusher decides it won, it'll "abort" pushee)